lib/fhmongodb.js

```diff
@@ -305,8 +305,15 @@ Database.prototype.remove = function (collectionName, id, callback) {
   this.db.collection(collectionName, function (err, collection) {
     if (null !== err) return callback(err, null);
 
+    // if a GUID id is passed as strings, convert it to an ObjectID 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)